### ISSUE TITLE
Questions about the results on Citeseer

### ISSUE THREAD

**Issue by nixror** [BUPT-GAMMA/CPF#4]
I tried to run the code on citeseer and use --automl to search best hyper-parameters , but only achieved 0.7276, far lower than the reported results. I used GCN as the teacher model and achieved 0.7110 on TITAN RTX GPU，which is the same as the result in the paper. But when I used RTX 2060 GPU, the result became different. 
So I have the following two questions:
1. Why I can't achieve the result reported in the paper on citeseer dataset even used the Optuna?  I would appreciate it if you could give the hyperparameters that can achieve the results of the paper.
2. Why the results are different when I run it on different GPUs？I checked the seed but I haven't found the reason yet.

**Comment by J-cabin** [comment]
1. We use heuristic search rather than Optuna in our experiments, and we provide Optuna only as a demonstration, so the searched hyperparameters may not be the best under default settings. In order to use Optuna to search for better hyperparameters, you need to read Optuna's documentation and modify the configuration parameters.
2. Different GPUs may indeed get different results, and the reasons may include (but are not limited to) different ways of handling floating-point numbers.
